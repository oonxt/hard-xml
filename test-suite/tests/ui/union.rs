use hard_xml::XmlRead;

#[derive(Clone, Copy)]
struct One {}

#[derive(XmlRead)]
union Data {
    one: One,
}

fn main() {}
