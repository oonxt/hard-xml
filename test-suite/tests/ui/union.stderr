error: hard-xml doesn't support union
 --> tests/ui/union.rs:7:1
  |
7 | / union Data {
8 | |     one: One,
9 | | }
  | |_^
